homeassistant/components/upnp/__init__.py

```diff
@@ -222,13 +223,29 @@ class UpnpEntity(CoordinatorEntity):
 
     coordinator: UpnpDataUpdateCoordinator
 
-    def __init__(self, coordinator: UpnpDataUpdateCoordinator) -> None:
+    def __init__(
+        self,
+        coordinator: UpnpDataUpdateCoordinator,
+        sensor_entity: EntityDescription,
+    ) -> None:
         """Initialize the base entities."""
         super().__init__(coordinator)
         self._device = coordinator.device
+        self._key = sensor_entity.key
```

Comment:
```suggestion
        self.entity_description = sensor_entity
```
Use this to override the Entity base class